python/mxnet/ndarray/contrib.py

```diff
@@ -28,7 +28,7 @@
 except ImportError:
     pass
 
-__all__ = ["rand_zipfian", "foreach", "while_loop"]
+__all__ = ["rand_zipfian", "foreach", "while_loop", "condition"]
```

Comment:
`condition` is not a good name. Maybe `cond`, or `conditional` or `if_else` or something like that